docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
```

Comment:
Is this required? Just wondering if the requirement for version 3 file format can be loosened

Reply:
Probably all the directives in this manifest would work on version 2 format but it's a way to enforce newer versions of Docker compose that are more secure and reliable

Reply:
Actually, from https://docs.docker.com/compose/compose-file/compose-versioning/#versioning 

> Note: When specifying the Compose file version to use, make sure to specify both the major and minor numbers. If no minor version is given, 0 is used by default and not the latest minor version.

Seems like 3.0 is already rather old, so there is no benefit in requiring that. I'll loosen up the requirement and just note which version I used for the demo

Reply:
`version` is required in the manifest so let's just pin to latest to ensure latest versions of docker and docker-compose